samples/cassandracluster.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   bootstrapImage: orangeopensource/cassandra-bootstrap:0.1.4
   configMapName: cassandra-configmap-v1
   dataCapacity: "200Mi"
-  dataStorageClass: "local-storage"
+  dataStorageClass: local-path
```

Comment:
WHere does this local-path come from? Is it an old update?

Reply:
K3d